### DISCUSSION TITLE
How to reference Home directory?

### DISCUSSION THREAD

**Discussion by rwong2888** [fastlane/fastlane#21467] [Q&A] [answered]
I am hard coding the home directory. I would like to use `~` or `$HOME` or something equivalent. How to accomplish this?

```
  sigh(
    output_path: "/Users/ec2-user/profiles",
  )
```

**Comment by victormarcel09** [comment]
To use the home directory in your code, you can use the environmental variable $HOME or the tilde (~) symbol. They both represent the home directory.
[paybyplatema](https://www.paybyplatema.me/)

**Comment by rwong2888** [comment, marked as answer]
`#{ENV["HOME"]}` was the solution
